### DISCUSSION TITLE
FizzBuzz( without for loop)

### DISCUSSION THREAD

**Discussion by tuna7588** [codewars/codewars.com#3186] [Show and tell]
Hi guys,
We just discuss about the FizzBuzz problem today in class and this is how we resolve it. I used to solve it with For loop, but today I use Map and forEach. I think this is just a great way to understand how these methods work. Here is the solution:

//getFunction
fizzBuzz = () => {
//getRangeOneToHundred
  let array = new Array(100).fill(0).map((_, index) => index + 1);
 
//getTransformArray
  let getTransformArray = array.map(number => {
    if (number % 15 === 0) number = 'FizzBuzz';
    if (number % 3 === 0) number = 'Fizz';
    if (number % 5 === 0) number = 'Buzz';
    return number;
  });
//getOutput
  return getTransformArray.forEach(number => console.log(number));
}
//print
fizzBuzz();

**Comment by hobovsky** [comment]
Please mind that Codewars Github is not meant for help with general programming or learning questions. It's meant to discuss matters related to Codewars platform, the website, its functions, etc.

To discuss interesting solutions and solving techniques, you can initiate a discussion under your solution on Codewars, or show it in `#beginners` sections of Codewars Discord.

I am going to close this discussion as off-topic for the Codewars Github.
